From f157f068bd42047e7318f3d1754d31d7dabc64f2 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Sat, 31 Mar 2012 14:25:23 +0800
Subject: [PATCH 1280/2463] ENGR00177235-2 SDMA: add p2p dma mode

Add code to support p2p dma mode.Add membership in imx_dma_data
struct to support P2P dma script. Because the P2P dma script
need 2 dma request to trigger DMA burst.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 arch/arm/plat-mxc/include/mach/dma.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/dma.h b/arch/arm/plat-mxc/include/mach/dma.h
index ee3b962..ba00959 100644
--- a/arch/arm/plat-mxc/include/mach/dma.h
+++ b/arch/arm/plat-mxc/include/mach/dma.h
@@ -55,6 +55,7 @@ enum imx_dma_prio {
 
 struct imx_dma_data {
 	int dma_request; /* DMA request line */
+	int dma_request_p2p;
 	enum sdma_peripheral_type peripheral_type;
 	int priority;
 };
-- 
1.7.7.4

